backend/coins/btc/account.go

```diff
@@ -496,14 +496,12 @@ func (account *Account) feeTargets() FeeTargets {
 	var feeTargets FeeTargets
 	if mempoolFees != nil {
 		feeTargets = FeeTargets{
-			{blocks: 12, code: accounts.FeeTargetCodeMempoolEconomy},
 			{blocks: 3, code: accounts.FeeTargetCodeMempoolHour},
 			{blocks: 2, code: accounts.FeeTargetCodeMempoolHalfHour},
 			{blocks: 1, code: accounts.FeeTargetCodeMempoolFastest},
 		}
 	} else {
 		feeTargets = FeeTargets{
-			{blocks: 24, code: accounts.FeeTargetCodeEconomy},
```

Comment:
I think this should not be removed right? This is the Bitcoin Core estimator if mempool.space is down.

Reply:
I think it would make sense to remove the Bitcoin Core economy option, for the same reasons. But if we want to do that, than we should clean it up completely, as done for the mempool one.